wforce/wforce-lua.cc

```diff
@@ -281,7 +282,7 @@ vector<std::function<void(void)>> setupLua(bool client, bool multi_lua, LuaConte
     c_lua.writeFunction("setSiblings", [](const vector<pair<int, string>>& parts) { });
   }
 
-  if (!multi_lua) {
+  if (!multi_lua && !client) {
```

Comment:
maybe || instead?

Reply:
Yeah maybe. I personally find !(xx || y) less readable than (!x && !y) but I think that's just how my brain works (i.e. strangely).

Reply:
well, no, i mean it seems like this should be

if not multi_lua or not client, then....

now it is

if not multi_lua *and* not client then...

Reply:
Ah yes, very good point! :)

Reply:
Also, Doh!

Reply:
So I've thought about this some more and we only want to define the non-stub function for the case where wforce is not a client AND multi-lua is not defined. So the original logic is correct.